collect_metadata/test_metadata.py

```diff
@@ -1,6 +1,10 @@
 # pylint: skip-file
 from unittest.mock import patch
+
 from extract import extract_botanist_data, extract_location_data
+from transform import turn_into_csv_input
+
+import requests as req
```

Comment:
Doesn't matter as much since it's a skipped test file, but third party imports should generally come before local ones